graphql-server/src/schemas/neo4j.ts

```diff
@@ -24,16 +19,6 @@ export async function buildNeo4jDbSchema() {
   const sessionFactory = () =>
     neo4jDriver.session({ defaultAccessMode: neo4j.session.WRITE });
 
-  const httpLink = createHttpLink({
-    uri: "https://localhost:4000/graphql",
-    fetch: fetch,
-  });
-
-  const client = new ApolloClient({
-    link: httpLink,
-    cache: new InMemoryCache(),
-  });
-
```

Comment:
Removed as these are not being used. (These were used previously to query some samples data as part of the old sample mutation workflow.)